src/components/form/form.scss

```diff
@@ -286,3 +286,14 @@ limel-code-editor {
 }
 
 @import './row/row.scss';
+
+limel-collapsible-section {
+    transition: transform 0.3s ease-in-out;
+
+    &.move-up-transition {
+        transform: translateY(-100%);
+    }
+    &.move-down-transition {
+        transform: translateY(100%);
+    }
```

Comment:
this only works when all collapsible sections have the same size. So for example if one is closed and one is open, this doesn't work and make a silly visual effect.